pkgs/applications/networking/instant-messengers/slack/default.nix

```diff
@@ -73,7 +73,8 @@ in stdenv.mkDerivation {
 
     # Fix the symlink
     rm $out/bin/slack
-    ln -s $out/lib/slack/slack $out/bin/slack
+    makeWrapper $out/lib/slack/slack $out/bin/slack \
+      --set LD_LIBRARY_PATH ${rpath}
```

Comment:
@zimbatm It wouldn't if you set dontPatchELF = true or do your patching in postFixup (I personally prefer the former). It is trimmed by patchelf --shrink-rpath in the fixup phase.